src/torznab.ts

```diff
@@ -210,7 +211,7 @@ async function createTorznabSearchQueries(
 			},
 		] as const;
 	} else if (mediaType === MediaType.SEASON && caps.tvSearch) {
-		const match = stem.match(SEASON_REGEX);
+		const match = stem.match(SEASON_REGEX) ?? stem.match(ARR_DIR_REGEX);
```

Comment:
What is the goal of `?? stem.match(ARR_DIR_REGEX)`? It doesn't contain any name groups which will always be undefined.

Reply:
Old code just left over....thanks